src/tex/output/.gitignore

```diff
@@ -3,3 +3,9 @@
 
 # Except the gitignore file itself!
 !.gitignore
+!count_extraction.txt
+!count_refusal.txt
+!extractions_files/
+!extractions_files/*
+!refusals_files/
+!refusals_files/*
```

Comment:
in principle, this rule is there because the stuff in `output` is supposed to be generated by `Snakemake` if it is not there. If you want to still share it, it might be worth to put those files into another place

Reply:
No, I introduced those rules to share them for revision. But after revision it has no sense to share them, specially when we are going to place a table with these results in the appendix